.github/workflows/check-labels.yml

```
@@ -17,6 +17,6 @@ jobs:
     steps:
       - uses: docker://onsdigital/github-pr-label-checker:v1.2.7
         with:
-          one_of: Breaking changes,Accessibility,Bug,Documentation,Dependencies,Enhancement,Example
+          one_of: Breaking changes,Accessibility,Bug,Documentation,Dependencies,Enhancement,Example,Component
```

If we are adding component we should also add pattern here
```suggestion
          one_of: Breaking changes,Accessibility,Bug,Documentation,Dependencies,Enhancement,Example,Component,Pattern
```